src/Viewer.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+class Viewer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      mainImg: props.phone.images[0],
+      description: props.phone.description,
+      name: props.phone.name,
+      images: props.phone.images,
+
+    };
+  }
+
+  handleShowImg(imgURl) {
+    this.setState(()=> {

```

Comment:
```suggestion
    this.setState({
      mainImg: imgURL
    })
```
you can simplify it by passing new state object directly to `setState`